src/components/result-view-tab.js

```diff
@@ -57,9 +59,18 @@ export const ResultViewTab = ({
     studyUuid,
     currentNode,
     openVoltageLevelDiagram,
+    resultTabIndexRedirection,
     disabled,
 }) => {
-    const [tabIndex, setTabIndex] = useState(0);
+    const [tabIndex, setTabIndex] = useState(
```

Comment:
don't use a separate state for tabIndex and resultTabIndexRedirection. It's duplicated state and needs extra rerenders to achieve the results